web/services/api.service.ts

```diff
@@ -31,8 +31,11 @@ export abstract class APIService {
     );
   }
 
-  get(url: string, params = {}) {
-    return this.axiosInstance.get(url, params);
+  get(url: string, params = {}, config = {}) {
+    return this.axiosInstance.get(url, {
```

Comment:
Why the extra param here?

Reply:
To fetch the binary data, we need to overwrite the default content type to-

```typescript
{
    "Content-Type": "application/octet-stream",
},
``` 

This is not accepted by the current `get` method.